tests/suite/manifests/ngf-upgrade/values.yaml

```diff
@@ -25,7 +25,3 @@ affinity:
       labelSelector:
         matchLabels:
           app.kubernetes.io/name: nginx-gateway
-
-service:
-  annotations:
-    networking.gke.io/load-balancer-type: "Internal"
```

Comment:
Why remove this?

Reply:
The tests already add this when installing NGF